docs/proposals/service-discovery/README.md

```diff
@@ -143,23 +149,33 @@ required) or even code snippets. If there's any ambiguity about HOW your
-4. Remote-Only Service Discovery: In cases where there is local service with the same name as the imported service, Karmada should remove the local `EndPointSlice` and create an `EndPointSlice` pointing to the remote cluster. This allows users to access the service through the imported `EndPointSlice`, effectively reaching the remote cluster.
+- The user creates a foo Deployment and Service on the Karmad control panel, and creates a PropagationPolicy to distribute them into the member cluster member1.
 
-Implementing the service registration flow, the following should be considered:
-1. Given that the `Service` named `foo` exists on cluster member1
-2. The `ServiceImport` resource is created on cluster member2, specifying the import of `foo`
-3. A `Service` resource is created in cluster member2, but does not prefix the name with `derived-`
-   1. If there is already an existing `Service` named `foo` on cluster member2
-   2. The `EndPointSlice` associated with the local `foo` service points to the local `Deployment` running on member2
-   3. The controller will ignore the conflict of existing `Service` and will not create a new `Service` object
-4. And the imported `EndPointSlice` is created, pointing to the remote `Deployment` running on member1
-5. And the `EndPointSlice` associated with the imported service is prefixed with `import-`
-6. Then the requests round-robin between the local `foo` service and the imported `foo` service (member1 and member2)
+![image](statics/user-operation-01.png)
+
+- The user creates an MCS object on the Karmada control plane to enable cross-cluster service foo. In this way, the service on cluster member2 can access the foo Service on cluster member1.
+
+![image](statics/user-operation-02.png)
+
+Then, present our specific plan design.
+
+- When the `mcs-controller` detects that a user has created a `MultiClusterService` object, it creates a `ServiceExport` in the Karmada control plane and propagates it to the source clusters through creating a `ResourceBinding` (the source clusters can obtain this via the Service associated with `ResourceBinding`).
+
+![image](statics/design-01.png)
+
+- Depending on the existing MCS atomic capabilities, the `service-export-controller` will collect the `EndpointSlices` related to `foo` Service into the Karmada control plane.
+
+![image](statics/design-02.png)
+
+- The `mcs-controller`, on the Karmada control plane, creates a `ResourceBinding` to propagate Service and EndpointSlice to destination clusters. This is done considering that some target Services already exist in certain destination clusters. Therefore, it's necessary to confirm the specific destination cluster based on the strategy specified in the `MultiClusterService` object.
```

Comment:
What is the specific behavior of eps syncing when there is or isn't a service in target cluster?

Reply:
If there is a Service existing on the target cluster, there is no need to resynchronize the EndpointSlices exported from this cluster to the cluster. Only synchronize the EndpointSlices received from other clusters. 

If there is no Service on the target cluster, both the Service and the EndpointSlices collected from other clusters need to be synchronized to that cluster.